src/store/ChartStore.js

```diff
@@ -614,10 +615,12 @@ class ChartStore {
     @action.bound newChart(symbolObj = this.currentActiveSymbol, params) {
         this.stxx.chart.symbolDisplay = symbolObj.name;
         this.loader.show();
+        this.mainStore.state.setChartStatus('loading');
```

Comment:
@nazaninreihani It'd be more efficient to invoke `setChartStatus` in `loader.show()` and `loader.hide()` functions instead of replicating it.

Reply:
we call loader.hide() conditionally and that is if loader exists, what if chart loads in a pace that no loader is rendered? :-D then the chart will not send the status